README.md

```diff
@@ -0,0 +1,68 @@
+# ScyllaDB JS Driver
+
+## Instalation and build process
+
+For instalation type command below:
+
+```
+$ npm install
+```
+
+For build process use this command:
+
+```
+$ npm run build
+```
+
+If command above don't work, make sure that you have napi package downloded globally.
+
+## File structure
+
+| File / Folder     | Origin                             | Usage                                  | Shouldn't be in .gitignore |
+| ----------------- | ---------------------------------- | -------------------------------------- | -------------------------- |
+| .cargo            | Generated by 'napi-rs'             | ?                                      | ?                          |
+| .github           | Generated by 'napi-rs'             | CI-CD                                  | ✅                         |
```

Comment:
What are the criteria for `Shouldn't be in .gitignore`? I don't immediately see any.

Reply:
"Shouldn't be in .gitignore" was column in which I was tracking if those files are auto-generated and should be in .gitignore to check if auto-generated .gitignore file was proper for our project. There is no longer a need for this column.